FamilyTree/po/it-local.po

```diff
@@ -1,7 +1,7 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Vincenzo Alfano <vinc.alfano@gmail.com>, YEAR.
```

Comment:
Please also the year (if you use a program like poedit or lokalize it should update those headers automatically)